its/src/test/java/com/sonar/it/csharp/UnitTestProjectTypeProbingTest.java

```diff
@@ -74,12 +74,7 @@ public void testProject_WithPropertySetToFalse_IsIdentifiedAsMainProject() {
 
   @Test
   public void msTestProject_IsIdentifiedAsMainProject() {
-    assertThat(getIssues("UTProjectProbing:UTProjectProbing.MsTest/calculator.cs"))
-      .extracting(Issues.Issue::getRule)
-      // After the Scanner for .NET gets updated to 5.2, this should be just MAIN_AND_TEST_RULE_ID.
-      // The Scanner for .NET 5.1 does not verify the project references.
-      // https://github.com/SonarSource/sonar-dotnet/issues/4169
-      .containsExactly(MAIN_AND_TEST_RULE_ID, MAIN_RULE_ID);
+    assertThat(getIssues("UTProjectProbing:UTProjectProbing.MsTest/calculator.cs")).isEmpty();
```

Comment:
The scanner has not been updated for a while so the test detection logic also changed in the meanwhile. Thanks to those changes now the .MsTest project is clarified as a test project as it has a reference to Microsoft.VisualStudio.TestPlatform.TestFramework.